From a1899fe3a0ca7a7920c9b273547e99e93724fe6b Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <nathan@kernel.org>
Date: Fri, 1 Jul 2022 09:14:38 -0700
Subject: [PATCH 05/10] HACK: Silence "error: write on a pipe with no reader"

Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 scripts/Makefile.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/Makefile.build b/scripts/Makefile.build
index 2a0521f77e5f..e459dac3f562 100644
--- a/scripts/Makefile.build
+++ b/scripts/Makefile.build
@@ -181,7 +181,7 @@ cmd_modversions_c =								\
 	fi;
 else
 cmd_modversions_c =								\
-	if $(OBJDUMP) -h $@ | grep -q __ksymtab; then				\
+	if $(OBJDUMP) -h $@ 2>/dev/null | grep -q __ksymtab; then			\
 		$(call cmd_gensymtypes_c,$(KBUILD_SYMTYPES),$(@:.o=.symtypes))	\
 		    > $(@D)/.tmp_$(@F:.o=.ver);					\
 										\
-- 
2.37.1

