From 0d67a5c526d4d82d506ba67a02b8bc336c682561 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <nathan@kernel.org>
Date: Mon, 28 Aug 2023 08:46:43 -0700
Subject: [PATCH 01/14] compiler-clang.h: __diag_clang_18()

Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 include/linux/compiler-clang.h | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/include/linux/compiler-clang.h b/include/linux/compiler-clang.h
index 9b673fefcef8..93adf2ac5201 100644
--- a/include/linux/compiler-clang.h
+++ b/include/linux/compiler-clang.h
@@ -119,6 +119,12 @@
 #define __diag_str(s)		__diag_str1(s)
 #define __diag(s)		_Pragma(__diag_str(clang diagnostic s))
 
+#if CONFIG_CLANG_VERSION >= 180000
+#define __diag_clang_18(s)	__diag(s)
+#else
+#define __diag_clang_18(s)
+#endif
+
 #if CONFIG_CLANG_VERSION >= 110000
 #define __diag_clang_11(s)	__diag(s)
 #else

base-commit: 2ee82481c392eec06a7ef28df61b7f0d8e45be2e
-- 
2.42.0

