From 25052ab5c3766ae81c9af44e34d99676fb2326b5 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Thu, 23 May 2019 13:00:45 -0700
Subject: [PATCH 01/18] Add localversion-cbl

Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 localversion-cbl | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 localversion-cbl

diff --git a/localversion-cbl b/localversion-cbl
new file mode 100644
index 000000000000..f434206111f4
--- /dev/null
+++ b/localversion-cbl
@@ -0,0 +1 @@
+-cbl

base-commit: 975f7a88c64dfdfde014530730ba7a6f3141f773
-- 
2.26.0

