From c4c439342223ff6bac43830889fc73ed3c1c5ecc Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <nathan@kernel.org>
Date: Wed, 1 Dec 2021 17:31:30 -0700
Subject: [PATCH 02/17] cachefiles: Apply diff of latest fscache-rewrite branch

Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 fs/cachefiles/io.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fs/cachefiles/io.c b/fs/cachefiles/io.c
index 74ef4d1fc562..60b1eac2ce78 100644
--- a/fs/cachefiles/io.c
+++ b/fs/cachefiles/io.c
@@ -485,8 +485,8 @@ static int __cachefiles_prepare_write(struct netfs_cache_resources *cres,
 
 	/* Partially allocated, but insufficient space: cull. */
 	fscache_count_no_write_space();
-	pos = cachefiles_inject_remove_error();
-	if (pos == 0)
+	ret = cachefiles_inject_remove_error();
+	if (ret == 0)
 		ret = vfs_fallocate(file, FALLOC_FL_PUNCH_HOLE | FALLOC_FL_KEEP_SIZE,
 				    *_start, *_len);
 	if (ret < 0) {
-- 
2.34.1

