From 6666144cd8d6f6c96fa663e8113065c50424061c Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <nathan@kernel.org>
Date: Tue, 7 Apr 2026 19:02:54 -0700
Subject: [PATCH 02/11] NOTCBL: WIP: fixup!    bufmap: manage as folios.

Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 fs/orangefs/orangefs-bufmap.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/orangefs/orangefs-bufmap.c b/fs/orangefs/orangefs-bufmap.c
index d476bd4df37c..59cebe483194 100644
--- a/fs/orangefs/orangefs-bufmap.c
+++ b/fs/orangefs/orangefs-bufmap.c
@@ -756,7 +756,7 @@ int orangefs_bufmap_copy_to_iovec(struct iov_iter *iter,
 	from = &__orangefs_bufmap->desc_array[buffer_index];
 
 	/* shouldn't happen... */
-	if (size > 4194304) pr_info("%s: size:%ld\n", __func__, size);
+	if (size > 4194304) pr_info("%s: size:%zu\n", __func__, size);
 
 	gossip_debug(GOSSIP_BUFMAP_DEBUG,
 		"%s: buffer_index:%d size:%zu folio_count:%d\n",
-- 
2.53.0

