From 3f05aa4eed17bbd7b1faa9cc07235382b30c5270 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <nathan@kernel.org>
Date: Fri, 8 Nov 2024 19:51:28 -0700
Subject: [PATCH 07/16] WIP: arch: Disable AutoFDO and Propeller when compile
 testing

Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 arch/Kconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/Kconfig b/arch/Kconfig
index 6682b2a53e34..9169afedd434 100644
--- a/arch/Kconfig
+++ b/arch/Kconfig
@@ -819,6 +819,7 @@ config AUTOFDO_CLANG
 	bool "Enable Clang's AutoFDO build (EXPERIMENTAL)"
 	depends on ARCH_SUPPORTS_AUTOFDO_CLANG
 	depends on CC_IS_CLANG && CLANG_VERSION >= 170000
+	depends on !COMPILE_TEST
 	help
 	  This option enables Clang’s AutoFDO build. When
 	  an AutoFDO profile is specified in variable
@@ -839,6 +840,7 @@ config PROPELLER_CLANG
 	bool "Enable Clang's Propeller build"
 	depends on ARCH_SUPPORTS_PROPELLER_CLANG
 	depends on CC_IS_CLANG && CLANG_VERSION >= 190000
+	depends on !COMPILE_TEST
 	help
 	  This option enables Clang’s Propeller build. When the Propeller
 	  profiles is specified in variable CLANG_PROPELLER_PROFILE_PREFIX
-- 
2.47.0

