From f89bbcbe6514afb8013ce8d47a1847403dfe47b2 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <nathan@kernel.org>
Date: Thu, 8 Sep 2022 06:55:52 -0700
Subject: [PATCH 07/23] NOTCBL: HACK: Export saved_command_line

Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 init/main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/init/main.c b/init/main.c
index 1c78bfc21142..5606d147d9dd 100644
--- a/init/main.c
+++ b/init/main.c
@@ -146,6 +146,7 @@ void (*__initdata late_time_init)(void);
 char __initdata boot_command_line[COMMAND_LINE_SIZE];
 /* Untouched saved command line (eg. for /proc) */
 char *saved_command_line;
+EXPORT_SYMBOL_GPL(saved_command_line);
 /* Command line for parameter parsing */
 static char *static_command_line;
 /* Untouched extra command line */
-- 
2.37.3

