From 7fe0a7eed5cc1cb9515bc50e74874266bc8ab8d4 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <nathan@kernel.org>
Date: Tue, 9 Feb 2021 16:25:02 -0700
Subject: [PATCH 10/15] max9271: Add MODULE_LICENSE

Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 drivers/media/i2c/max9271.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/media/i2c/max9271.c b/drivers/media/i2c/max9271.c
index c247db569bab..a56bfdbe737a 100644
--- a/drivers/media/i2c/max9271.c
+++ b/drivers/media/i2c/max9271.c
@@ -18,9 +18,12 @@
 
 #include <linux/delay.h>
 #include <linux/i2c.h>
+#include <linux/module.h>
 
 #include "max9271.h"
 
+MODULE_LICENSE("GPL");
+
 static int max9271_read(struct max9271_device *dev, u8 reg)
 {
 	int ret;
-- 
2.30.1

