From 95d185a9ef6020897052c02a63e5ef232ec0fc25 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Sat, 13 Jun 2020 23:01:08 -0700
Subject: [PATCH 14/16] imx-mailbox: Shut up unused function warnings

Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 drivers/mailbox/imx-mailbox.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/mailbox/imx-mailbox.c b/drivers/mailbox/imx-mailbox.c
index 7205b825c8b5..dde6d4f30c57 100644
--- a/drivers/mailbox/imx-mailbox.c
+++ b/drivers/mailbox/imx-mailbox.c
@@ -598,6 +598,7 @@ static const struct of_device_id imx_mu_dt_ids[] = {
 };
 MODULE_DEVICE_TABLE(of, imx_mu_dt_ids);
 
+#ifdef CONFIG_PM_SLEEP
 static int imx_mu_suspend_noirq(struct device *dev)
 {
 	struct imx_mu_priv *priv = dev_get_drvdata(dev);
@@ -625,7 +626,9 @@ static int imx_mu_resume_noirq(struct device *dev)
 
 	return 0;
 }
+#endif
 
+#ifdef CONFIG_PM
 static int imx_mu_runtime_suspend(struct device *dev)
 {
 	struct imx_mu_priv *priv = dev_get_drvdata(dev);
@@ -646,6 +649,7 @@ static int imx_mu_runtime_resume(struct device *dev)
 
 	return ret;
 }
+#endif
 
 static const struct dev_pm_ops imx_mu_pm_ops = {
 	SET_NOIRQ_SYSTEM_SLEEP_PM_OPS(imx_mu_suspend_noirq,
-- 
2.27.0

