From 60e09eb8eb871560e5952279d90ec4ba06ba5528 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Thu, 9 Jan 2020 12:33:20 -0700
Subject: [PATCH 17/22] qcom-cpr: -Wsometimes-uninitialized

Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 drivers/power/avs/qcom-cpr.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/power/avs/qcom-cpr.c b/drivers/power/avs/qcom-cpr.c
index 9192fb747653..902756f2b7e9 100644
--- a/drivers/power/avs/qcom-cpr.c
+++ b/drivers/power/avs/qcom-cpr.c
@@ -460,8 +460,8 @@ static unsigned int cpr_get_cur_perf_state(struct cpr_drv *drv)
 
 static int cpr_scale(struct cpr_drv *drv, enum voltage_change_dir dir)
 {
-	u32 val, error_steps, reg_mask;
-	int last_uV, new_uV, step_uV, ret;
+	u32 val, error_steps, reg_mask = 0;
+	int last_uV, new_uV = 0, step_uV, ret;
 	struct corner *corner;
 	const struct cpr_desc *desc = drv->desc;
 
-- 
2.25.0

