From 805a0f85587c430aaafcd004bc7ed329ddc261f8 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Sat, 25 Jan 2020 22:49:02 -0700
Subject: [PATCH 19/19] -Wfortify-source

Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 kernel/printk/printk.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/kernel/printk/printk.c b/kernel/printk/printk.c
index fada22dc4ab6..cca579e77f77 100644
--- a/kernel/printk/printk.c
+++ b/kernel/printk/printk.c
@@ -2412,6 +2412,7 @@ void console_unlock(void)
 
 		printk_safe_enter_irqsave(flags);
 		raw_spin_lock(&logbuf_lock);
+#ifdef CONFIG_PRINTK
 		if (console_seq < log_first_seq) {
 			len = sprintf(text,
 				      "** %llu printk messages dropped **\n",
@@ -2420,9 +2421,9 @@ void console_unlock(void)
 			/* messages are gone, move to first one */
 			console_seq = log_first_seq;
 			console_idx = log_first_idx;
-		} else {
+		} else
+#endif
 			len = 0;
-		}
 skip:
 		if (console_seq == log_next_seq)
 			break;
-- 
2.25.0

