From 803c10cf5fe6ed9024b6aacbda9958f643afb444 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Tue, 5 Nov 2019 21:39:07 -0700
Subject: [PATCH 21/23] ext4: Label sbi maybe unused in parse_options

Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 fs/ext4/super.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/ext4/super.c b/fs/ext4/super.c
index f3279210f0ba..ee8c42d8a04f 100644
--- a/fs/ext4/super.c
+++ b/fs/ext4/super.c
@@ -2065,7 +2065,7 @@ static int parse_options(char *options, struct super_block *sb,
 			 unsigned int *journal_ioprio,
 			 int is_remount)
 {
-	struct ext4_sb_info *sbi = EXT4_SB(sb);
+	struct ext4_sb_info __maybe_unused *sbi = EXT4_SB(sb);
 	char *p, __maybe_unused *usr_qf_name, __maybe_unused *grp_qf_name;
 	substring_t args[MAX_OPT_ARGS];
 	int token;
-- 
2.24.0

