From bd9f6fea81f83c11523f3347dbaa75e2a0ab1c74 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Thu, 16 Jan 2020 17:11:54 -0700
Subject: [PATCH 21/21] x86: Disable bc7c9d62052b32b1f1fe5a141165a0f4d82103ac
 for clang

Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 arch/x86/boot/compressed/eboot.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/x86/boot/compressed/eboot.c b/arch/x86/boot/compressed/eboot.c
index 4afd29eb5b34..e3fb8e90d351 100644
--- a/arch/x86/boot/compressed/eboot.c
+++ b/arch/x86/boot/compressed/eboot.c
@@ -6,7 +6,9 @@
  *
  * ----------------------------------------------------------------------- */
 
+#ifndef CONFIG_CC_IS_CLANG
 #pragma GCC visibility push(hidden)
+#endif
 
 #include <linux/efi.h>
 #include <linux/pci.h>
-- 
2.25.0

