From 1dd83f46b91d66678421ff9f62a54bdbe362be2f Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Wed, 4 Dec 2019 00:42:56 -0700
Subject: [PATCH 22/22] fs/quota: Remove unused label

Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 fs/quota/dquot.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/fs/quota/dquot.c b/fs/quota/dquot.c
index d6bf9bf3988c..b0688c02dc90 100644
--- a/fs/quota/dquot.c
+++ b/fs/quota/dquot.c
@@ -2496,7 +2496,6 @@ int dquot_quota_on_mount(struct super_block *sb, char *qf_name,
 		error = dquot_load_quota_inode(d_inode(dentry), type, format_id,
 				DQUOT_USAGE_ENABLED | DQUOT_LIMITS_ENABLED);
 
-out:
 	dput(dentry);
 	return error;
 }
-- 
2.24.0

