From 80adf06979731f7041c9fa9894e8c22232f3af77 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Wed, 7 Nov 2018 21:46:39 -0700
Subject: [PATCH 6/6] sunvdc: Removed unused variable 'ret'

Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 drivers/block/sunvdc.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/block/sunvdc.c b/drivers/block/sunvdc.c
index 95cb4ea8e402..175a64619602 100644
--- a/drivers/block/sunvdc.c
+++ b/drivers/block/sunvdc.c
@@ -785,7 +785,6 @@ static void cleanup_queue(struct request_queue *q)
 static struct request_queue *init_queue(struct vdc_port *port)
 {
 	struct request_queue *q;
-	int ret;
 
 	q = blk_mq_init_sq_queue(&port->tag_set, &vdc_mq_ops, VDC_TX_RING_SIZE,
 					BLK_MQ_F_SHOULD_MERGE);
-- 
2.19.1

