From 063b990c6311057e78b36f4c02a0f8ad6007ddd1 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Thu, 23 May 2019 13:00:45 -0700
Subject: [PATCH 01/23] Add localversion-cbl

Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 localversion-cbl | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 localversion-cbl

diff --git a/localversion-cbl b/localversion-cbl
new file mode 100644
index 000000000000..f434206111f4
--- /dev/null
+++ b/localversion-cbl
@@ -0,0 +1 @@
+-cbl

base-commit: d3dca69085e94e52a1d61a34b8e5f73a9f3d7eed
-- 
2.26.0.rc1

