From b31226bc2eb7edbf1af847f9985055a824faf3f3 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Thu, 23 May 2019 13:00:45 -0700
Subject: [PATCH 01/19] Add localversion-cbl

Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 localversion-cbl | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 localversion-cbl

diff --git a/localversion-cbl b/localversion-cbl
new file mode 100644
index 000000000000..f434206111f4
--- /dev/null
+++ b/localversion-cbl
@@ -0,0 +1 @@
+-cbl

base-commit: fd6988496e79a6a4bdb514a4655d2920209eb85d
-- 
2.24.1

