From 7206ac43aecbd9b286a148b243039457ad301351 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Thu, 23 May 2019 13:00:45 -0700
Subject: [PATCH 01/19] Add localversion-cbl

Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 localversion-cbl | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 localversion-cbl

diff --git a/localversion-cbl b/localversion-cbl
new file mode 100644
index 000000000000..f434206111f4
--- /dev/null
+++ b/localversion-cbl
@@ -0,0 +1 @@
+-cbl

base-commit: 6794862a16ef41f753abd75c03a152836e4c8028
-- 
2.24.0

