From 34cc4e1e674b7f917fa6e15e4f8e257fa794aeaf Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <nathan@kernel.org>
Date: Wed, 10 Jul 2024 10:31:49 -0700
Subject: [PATCH 5/7] HACK: LONGTERM: Remove clone3() #warning from
 kernel/fork.c

Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 kernel/fork.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/kernel/fork.c b/kernel/fork.c
index 4fc3611435501..886ff79292cd1 100644
--- a/kernel/fork.c
+++ b/kernel/fork.c
@@ -3109,7 +3109,6 @@ SYSCALL_DEFINE2(clone3, struct clone_args __user *, uargs, size_t, size)
 	pid_t set_tid[MAX_PID_NS_LEVEL];
 
 #ifdef __ARCH_BROKEN_SYS_CLONE3
-#warning clone3() entry point is missing, please fix
 	return -ENOSYS;
 #endif
 
-- 
2.49.0

