From d20a30f4c1f7498a0b68ac5a4e0da562a160dcf7 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Mon, 10 Dec 2018 21:00:32 -0700
Subject: [PATCH 04/11] DO-NOT-UPSTREAM: Disable -Wframe-larger-than= for Clang
 on arm32

Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 lib/Kconfig.debug | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/Kconfig.debug b/lib/Kconfig.debug
index f8351f1e07d6..8d43075e6947 100644
--- a/lib/Kconfig.debug
+++ b/lib/Kconfig.debug
@@ -242,6 +242,7 @@ config FRAME_WARN
 	default 0 if KASAN && CC_IS_CLANG
 	default 2048 if GCC_PLUGIN_LATENT_ENTROPY
 	default 1280 if (!64BIT && PARISC)
+	default 0 if (!64BIT && !PARISC && CC_IS_CLANG)
 	default 1024 if (!64BIT && !PARISC)
 	default 2048 if 64BIT
 	help
-- 
2.20.1

