From 385acab9bd5fd4c9f2e7115b2c630f957726b5c4 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <nathan@kernel.org>
Date: Wed, 10 Jul 2024 10:31:49 -0700
Subject: [PATCH 12/13] HACK: LONGTERM: Remove clone3() #warning from
 kernel/fork.c

Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 kernel/fork.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/kernel/fork.c b/kernel/fork.c
index a01cf3a904bf..5043bf4c59a0 100644
--- a/kernel/fork.c
+++ b/kernel/fork.c
@@ -3109,7 +3109,6 @@ SYSCALL_DEFINE2(clone3, struct clone_args __user *, uargs, size_t, size)
 	pid_t set_tid[MAX_PID_NS_LEVEL];
 
 #ifdef __ARCH_BROKEN_SYS_CLONE3
-#warning clone3() entry point is missing, please fix
 	return -ENOSYS;
 #endif
 
-- 
2.54.0

