From 117e9ac2a41f97bd2fdc0bf17cdd1d6622ecaf63 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <nathan@kernel.org>
Date: Fri, 1 Jul 2022 09:14:38 -0700
Subject: [PATCH 08/11] HACK: Silence "error: write on a pipe with no reader"

Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 scripts/Makefile.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/Makefile.build b/scripts/Makefile.build
index 17aa8ef2d52a..2b91fb6f5e42 100644
--- a/scripts/Makefile.build
+++ b/scripts/Makefile.build
@@ -182,7 +182,7 @@ cmd_modversions_c =								\
 	fi;
 else
 cmd_modversions_c =								\
-	if $(OBJDUMP) -h $@ | grep -q __ksymtab; then				\
+	if $(OBJDUMP) -h $@ 2>/dev/null | grep -q __ksymtab; then			\
 		$(call cmd_gensymtypes_c,$(KBUILD_SYMTYPES),$(@:.o=.symtypes))	\
 		    > $(@D)/.tmp_$(@F:.o=.ver);					\
 										\
@@ -376,7 +376,7 @@ ifdef CONFIG_ASM_MODVERSIONS
 # we parse asm-prototypes.h C header to get function definitions.
 
 cmd_modversions_S =								\
-	if $(OBJDUMP) -h $@ | grep -q __ksymtab; then				\
+	if $(OBJDUMP) -h $@ 2>/dev/null | grep -q __ksymtab; then			\
 		$(call cmd_gensymtypes_S,$(KBUILD_SYMTYPES),$(@:.o=.symtypes))	\
 		    > $(@D)/.tmp_$(@F:.o=.ver);					\
 										\
-- 
2.38.1

