From 93992dc370897394f38f31eb5bbd35f783342f4a Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <nathan@kernel.org>
Date: Wed, 20 Oct 2021 15:21:16 -0700
Subject: [PATCH 11/11] HACK: fs/namei.c: -Wbitwise-instead-of-logical

Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 fs/namei.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/namei.c b/fs/namei.c
index 2ea15d043412..74771c98bc92 100644
--- a/fs/namei.c
+++ b/fs/namei.c
@@ -2146,7 +2146,7 @@ static inline u64 hash_name(const void *salt, const char *name)
 inside:
 		a = load_unaligned_zeropad(name+len);
 		b = a ^ REPEAT_BYTE('/');
-	} while (!(has_zero(a, &adata, &constants) | has_zero(b, &bdata, &constants)));
+	} while (!((unsigned long)has_zero(a, &adata, &constants) | has_zero(b, &bdata, &constants)));
 
 	adata = prep_zero_mask(a, adata, &constants);
 	bdata = prep_zero_mask(b, bdata, &constants);
-- 
2.37.1

