From 70ad98f3db21430b243ac35984d1521222b2d6fd Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <nathan@kernel.org>
Date: Mon, 16 Dec 2024 17:41:11 -0700
Subject: [PATCH 23/24] HACK: modpost: Ignore references from the CFI jumptable
 section

Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 scripts/mod/modpost.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/scripts/mod/modpost.c b/scripts/mod/modpost.c
index 891930205ee9..eee4371fc0ee 100644
--- a/scripts/mod/modpost.c
+++ b/scripts/mod/modpost.c
@@ -939,6 +939,7 @@ static const char *const data_sections[] = { DATA_SECTIONS, NULL };
 	"*_console"
 
 static const char *const head_sections[] = { ".head.text*", NULL };
+static const char *const cfi_jt_sections[] = { ".text..L.cfi.jumptable*", NULL };
 static const char *const linker_symbols[] =
 	{ "__init_begin", "_sinittext", "_einittext", NULL };
 static const char *const optim_symbols[] = { "*.constprop.*", NULL };
@@ -1213,6 +1214,10 @@ static int secref_whitelist(const struct sectioncheck *mismatch,
 	if (strstarts(fromsym, ".L"))
 		return 0;
 
+	if (match(fromsec, cfi_jt_sections) &&
+	    match(tosec, init_exit_sections))
+		return 0;
+
 	return 1;
 }
 
-- 
2.53.0

